woocommerce/woocommerce-gateway-stripe
# [UPE - settings] Add capabilities messaging for payment methods

# Changes proposed in this Pull Request:

Resolves #1893 

Considering the Stripe account capabilities, this PR is adding a message for each payment method in the payment methods table when those are not active on that Stripe account. It's under the same scope of the "requires currency" message.

# Testing instructions

Make sure you have a **non** active account while testing, otherwise all the payment methods will be active and the message won't display.

1. Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe (Log in as an admin).
2. If any of the payment methods is not active, you should see a message "Pending activation" with a tooltip next to the payment method name. If the "Requires currency" applies, both messages should be visible on that payment method.

-------------------
